engines/sci/resource/resource.cpp

```diff
@@ -3122,6 +3122,28 @@ Common::String ResourceManager::findSierraGameId(const bool isBE) {
 	return heap->getStringAt(offset);
 }
 
+// Mac executables are currently used for icon bars and native fonts.
+// Eventually they should be used for native menus and possibly even splash screens.
+// For example, LSL6 can't function without native menus. (bug #11356)
+// Executables that we currently don't use are commented out.
+Common::String ResourceManager::getMacExecutableName() const {
+	switch (g_sci->getGameId()) {
+	case GID_CASTLEBRAIN: return "Castle of Dr. Brain"; // fonts
+	case GID_FREDDYPHARKAS: return "Freddy Pharkas"; // fonts, icon bar
+	//case GID_HOYLE4: return "Hoyle"; // doesn't use native fonts
```

Comment:
So, does this game have a Mac icon bar, and does it use Mac menus? Please enhance the comment here (same for SQ4 below)

Reply:
I finished inventorying all the executables; this is now updated with what each one contains